packages/ibm-products/src/components/Tearsheet/TearsheetShell.tsx

```diff
@@ -550,6 +550,9 @@ export const TearsheetShell = React.forwardRef(
                     alwaysRender={
                       !!(influencer && influencerPosition === 'right')
                     }
+                    {...{
+                      tabIndex: -1,
```

Comment:
instead of trying to bypass the typescript check here i think we should just include `tabIndex` in the type definition